pkgs/fabric.yaml

```diff
@@ -4,5 +4,16 @@ dependencies:
   run: [paramiko]
 
 sources:
-  - url: https://pypi.python.org/packages/source/F/Fabric/Fabric-1.8.0.tar.gz
-    key: tar.gz:ab3mo2v654775els4euq43sipyr2oiu3
+- key: tar.gz:mzvd3kjconjs2osj3unmkpr5oiof5qxs
+  url: https://pypi.python.org/packages/source/F/Fabric/Fabric-1.10.2.tar.gz
+
+build_stages:
+
+- name: install
+  after: setup_dirs
+  handler: bash
+  bash: |
```

Comment:
To be honest, I didn't spend much time investigating, it was what worked for me at the time. I think this has arisen a few times for me in the past, I previously just swapped out setuptools for distutils or vice versa and the issue goes away.